admin/taxonomy/class-taxonomy-metabox.php

```diff
@@ -54,144 +56,118 @@ class WPSEO_Taxonomy_Metabox {
 	public function __construct( $taxonomy, $term ) {
 		$this->term                 = $term;
 		$this->taxonomy             = $taxonomy;
-		$this->taxonomy_tab_content = new WPSEO_Taxonomy_Fields_Presenter( $this->term );
+		$this->social_is_enabled = WPSEO_Options::get( 'opengraph', false ) || WPSEO_Options::get( 'twitter', false );
```

Comment:
With booleans it's conventional to start with the noun, so I would suggest `is_social_enabled`.